app/models.py

```diff
@@ -550,3 +557,21 @@ def bulk_lookup():
                 db.session.commit()
             except IntegrityError:
                 db.session.rollback()
+
+class Leaders(db.Model):
```

Comment:
If you're keen to keep the model, I'd try hacking the migrations themselves - in upgrade() I'd suggest you try replacing the generated code (that creates the table) with a call to sql/views/leaders.sql (rather than doing it in manage.py).